### DISCUSSION TITLE
Is it possible to replace italic with bold in kitty whenever possible?

### DISCUSSION THREAD

**Discussion by c02y** [kovidgoyal/kitty#7872] [Q&A]
In kitty, some words are displayed as italic, I don't like the italic style of my font, is it possible to replace all the italic style with bold, which Alacritty is doing?

my font:
![image](https://github.com/user-attachments/assets/b328ec95-10d5-42f0-a70b-8d5cbc4519e8)

Kitty vs Alacritty about italic and bold:
![image](https://github.com/user-attachments/assets/e25c369a-1de0-4e06-be50-1c16f3678c55)

font config in my kitty.conf:
```
font_family      OperatorMono Nerd Font Mono
bold_font        auto
italic_font      auto
bold_italic_font auto
```

FYI:
kitty 0.36.2-1
kitty-shell-integration 0.36.2-1
kitty-terminfo 0.36.2-1
Archlinux: Linux 6.10.10-zen1-1-zen

**Comment by kovidgoyal** [comment]
Update kitty, run the choose-fonts kitten choose whatever face you like
for any given style.
